ember_debug/adapters/web-extension.js

```diff
@@ -1,10 +1,16 @@
 import BasicAdapter from "./basic";
 const Ember = window.Ember;
-const { computed, run, typeOf } = Ember;
+const { run, typeOf } = Ember;
 const { isArray } = Array;
 const { keys } = Object;
 
 export default BasicAdapter.extend({
+  init() {
+    this._super(...arguments);
+    this.set('_channel', new MessageChannel());
+    this.set('_chromePort', this.get('_channel.port1'));
```

Comment:
I doubled check that no code is setting these two variables and since we expect variables starting with `_` we should be good?

Reply:
I agree 👍